index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-session">
+      <h2>Login</h2>
+      <h3>Enter your credentials</h3>
+      <a href="./register.html" id="register-html">Register here</a>
```

Comment:
The register link is more appropriate at the bottom. I think it will sit better below the "Forgot password" link.